### DISCUSSION TITLE
Subscription to a large feed length

### DISCUSSION THREAD

**Discussion by One-Qiqirn** [FreshRSS/FreshRSS#6865] [Q&A] [answered]
I want to subscribe to this RSS feed, but I can't because of restrictions. Is there a way around this?

https://security.netapp.com/advisory/index.xml

Verification results for https://validator.w3.org/feed/
>Unable to validate, due to hardcoded resource limits (feed length > 5000000 bytes)

**Comment by Frenzie** [comment]
If you click advanced you can choose a different curl download timeout, or you can set a higher PHP processing timeout in php.ini.

**Comment by Alkarex** [comment, marked as answer]
Hello,
Such a large feed requires more memory for parsing. Allowing more memory to PHP depends on your setup (did you install using Docker, git, ... ?), but one easy way that should work in most cases is to add a line like

```php
<?php
ini_set('memory_limit', '256M');
```

to your `./Freshrss/constants.local.php` file (create it if it does not exist)

**Reply by Frenzie** [reply]
It's also large enough and slow enough that it's likely to exceed the default (?) curl timeout of 10 s fairly frequently.

**Reply by Alkarex** [reply]
20s by default, but yes, might be a problem on slow connections (9-12s for my tests)

**Reply by One-Qiqirn** [reply]
> <?php
> ini_set('memory_limit', '256M');
Thank you for your reply.
I was able to subscribe thanks to you.
